indexing-service/src/main/java/io/druid/indexing/overlord/supervisor/SupervisorResource.java

```diff
@@ -82,13 +81,18 @@ public Response specPost(final SupervisorSpec spec, @Context final HttpServletRe
           public Response apply(SupervisorManager manager)
           {
             if (authConfig.isEnabled()) {
-              // This is an experimental feature, see - https://github.com/druid-io/druid/pull/2424
-              final AuthorizationInfo authorizationInfo = (AuthorizationInfo) req.getAttribute(AuthConfig.DRUID_AUTH_TOKEN);
-              Preconditions.checkNotNull(
-                  authorizationInfo,
-                  "Security is enabled but no authorization info found in the request"
+              Preconditions.checkArgument(
```

Comment:
this used to be in `checkSupervisorAccess` which I removed, that was called within the if(authConfig.enabled()) block previously